nova/modules/core/libraries/Nova_location.php

```diff
@@ -255,15 +255,29 @@ public static function view($view, $skin, $section, $data = null)
 			$location = $obj->skin.'/'.$obj->sec.'/pages/'.$obj->view;
 		}
 		
+		$ci->event->trigger(['location', 'view', 'data', $obj->sec, $obj->view], [
```

Comment:
I'm actually not a big fan of this API.

I'd like to first understand why we're passing an array of values in and then figure out if there's a more eloquent way to handle something like this.